src/test/java/org/openrewrite/java/security/xml/TransformerFactoryXXEVulnerabilityTest.java

```diff
@@ -0,0 +1,679 @@
+/*
+ * Copyright 2022 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.security.xml;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.java.security.XmlParserXXEVulnerability;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+class TransformerFactoryXXEVulnerabilityTest implements RewriteTest {
```

Comment:
Should we rename this class _and file_ to match the recipe it tests?
```suggestion
class XmlParserXXEVulnerabilityTest implements RewriteTest {
```

Reply:
There is already a `XmlParserXXEVulnerabilityTest`

Reply:
This test is actually testing an individual visitor, which the recipe happens to invoke. Instead, we chose to rename the `XmlInputFactoryXXEVulnerabilityTest`